spec/takeaway_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'takeaway'
+
+describe Takeaway do 
+	subject(:takeaway) {described_class.new}
+
+  context '#menu' do
+  	it "has a class Takeaway" do
+  		expect(takeaway).to be_an_instance_of Takeaway
+  	end
+
+  	it "has a menu (list) of dishes" do
+      expect(takeaway.menu).to include("Burger")
+    end
```

Comment:
Lint/BlockAlignment: `end` at 13, 4 is not aligned with `it "has a menu (list) of dishes" do` at 11, 3.